tests/test_main.py

```diff
@@ -327,9 +327,9 @@ def test_elves_concert(elves, songs):
         ),
         (
                 [
-                    DwarfWarrior(nickname="Thiddeal", favourite_dish="French Fries", hummer_level=3),
-                    DwarfWarrior(nickname="Dwarf", favourite_dish="Caesar Salad", hummer_level=3),
-                    DwarfWarrior(nickname="Dwarf2", favourite_dish="French Fries", hummer_level=3),
+                    DwarfWarrior(nickname="Thiddeal", _favourite_dish="French Fries", hummer_level=3),
+                    DwarfWarrior(nickname="Dwarf", _favourite_dish="Caesar Salad", hummer_level=3),
+                    DwarfWarrior(nickname="Dwarf2", _favourite_dish="French Fries", hummer_level=3),
                 ],
                 (
                         "Thiddeal is eating French Fries\n"
```

Comment:
Revert your changes in tests and make appropriate changes in the signature of the `__init__` of the underlying classes. You should take an argument without `_` and later write its value to the instance attribute like `self._...`.